src/main/splitting/splitting.py

```diff
@@ -71,21 +71,36 @@ def find_task_dfs(df: pd.DataFrame, task: consts.TASK) -> List[pd.DataFrame]:
     return [split_df[split_df[CHOSEN_TASK] == task.value] for split_df in split_dfs]
 
 
-# 2.0 version with a different task_df extraction
+def __get_dst_path(src_file: str, output_directory: str) -> str:
+    file_name = get_name_from_path(src_file)
+    task_path = get_parent_folder(src_file)
+    task = get_name_from_path(task_path, with_extension=False)
+    language = get_parent_folder_name(task_path)
+    return os.path.join(output_directory, language, task, file_name)
+
+
+# 3.0 version: reorganize the file structure
+# Before the function calling:
+# -root
+# --user_N1
+# ---task1
+# ----user_N1_files
+# --user_N2
+# ---task1
+# ----user_N2_files
+# After the function calling:
+# -root
+# --task1
+# ----user_N1_files
+# ----user_N2_files
+# Todo: I am not sure about this name
 def split_tasks_into_separate_files(path: str, output_directory_suffix: str = 'separated_tasks') -> str:
```

Comment:
Let's call it 'reorganize_files_structure'? And maybe keep the previous function just in case as a separate one?

Reply:
I think we can find the previous version in the commits history if we want